pkgs/development/libraries/openssl/default.nix

```diff
@@ -172,7 +174,7 @@ let
       mkdir -p $bin
       mv $out/bin $bin/bin
 
-    '' + lib.optionalString (!stdenv.hostPlatform.isWindows)
+    '' + lib.optionalString (makeWrapper != null)
```

Comment:
`lib.meta.availableOn makeWrapper stdenv.hostPlatform` would be a possibility or querying `meta.broken` of `makeWrapper`—or just expanding the conditional.

For better or worse, the convention in nixpkgs is that the expression using a package decides whether it can use it or not. This also has the sense that it prevents stuff from being silently unavailable which can alter configure time settings etc.